src/config/graphical-session/wayland.md

```diff
@@ -32,18 +32,22 @@ Void Linux currently packages the following Wayland compositors:
 Both GNOME and KDE Plasma have EGLStreams backends for Wayland, which means they
 can use the proprietary NVIDIA drivers. Most other Wayland compositors require
 drivers that implement the GBM interface. The main driver for this purpose is
-provided by the `mesa-dri` package. The [graphics
+provided by the `mesa-dri` package. The [graphics drivers
```

Comment:
This should use [the double-quoting style for referring to section titles](https://docs.voidlinux.org/contributing/void-docs/style-guide.html#internal-links).